test/e2e/space_autocompletion_test.go

```diff
@@ -54,11 +56,11 @@ func TestAutomaticClusterAssignment(t *testing.T) {
 		time.Sleep(1 * time.Second)
 
 		// when
-		space1, _ := CreateSpaceWithBinding(t, awaitilities, mur, WithName("space-waitinglist1"))
+		space1, _ := CreateSpaceWithBinding(t, awaitilities, mur, testspace.WithName("space-waitinglist1"), testspace.WithTierName("appstudio"))
```

Comment:
Are those `testspace.WithName` being actually used ? I'm asking because seems like the new `CreateSpaceWithBinding` function "forces" the name of the space to be generated from the name of the test : https://github.com/codeready-toolchain/toolchain-e2e/pull/758/files#diff-c506d474c3fa798d8fc4c16ac5589eba599f306a438d1f2ea5018a770bbdd880R65-R66 

but maybe I've missed something.

Reply:
no worries, you're welcome!